webui/react/src/components/ResourcePoolCard.tsx

```diff
@@ -125,13 +126,15 @@ const ResourcePoolCard: React.FC<Props> = ({ resourcePool: pool }: Props) => {
           {pool.description && <Icon name="info" title={pool.description} />}
         </div>
       </div>
-      <div className={css.body}>
-        <RenderAllocationBarResourcePool resourcePool={pool} size={ShirtSize.Medium} />
-        <section className={css.details}>
-          <Json hideDivider json={shortDetails} />
-        </section>
-        <div />
-      </div>
+      <Suspense fallback={<Spinner center />}>
```

Comment:
Is this `Suspense` actually working?
I couldnt see `Spinner`

Reply:
It is. When I was testing the loading states were so brief that I had to manually trigger them by adding `Loadable.waitFor(NotFound)` to see them.

Reply:
I tested it with `fast 3G` and `slow 3G`, but I didn't see spinner. 
Could you tell me how you tested with `Loadable.waitFor(NotFound)`? Prob im doing wrong